src/xdr/Stellar-overlay.x

```diff
@@ -112,7 +112,7 @@ case DONT_HAVE:
 case GET_PEERS:
     void;
 case PEERS:
-    PeerAddress peers<>;
```

Comment:
that does not change the binary format?

Reply:
no change in the binary format, the `<N>` notation only changes the marshaller to enforce a max